### PR TITLE
feature(monorepo): adding 1password secrets

### PR DESCRIPTION
**Pull request by julien51**
# Description

Finally exploring better ways to do secret in our apps! This is a prereq to #12158  since we will have so many new secrets!

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -4,6 +4,20 @@ on:
...
+  load-secrets:
+    steps:
+      - name: Load secrets from 1Password
+        uses: 1Password/load-secrets-action@v1.2.0
+        with:
+          export-env: true
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          USERNAME: op://API & Services/hello-world/username
+          CREDENTIAL: op://API & Services/hello-world/credential
```
Comment (searchableguy) [inline .github/workflows/pull-request.yml:16]: this is placeholder?

Comment (julien51) [reply .github/workflows/pull-request.yml:16]: yes :)

**Comment by julien51** [comment]
That worked!

Screenshot 2023-06-13 at 6 27 34 PM

**Comment by julien51** [comment]
Now the next step is to bring all the secrets to 1Password.

**Comment by julien51** [comment]
We also need to set the heroku env vars from here...

**Review by clemsos — APPROVED** [review]
LG! Mounting a valut seems a much easier way to handle secrets indeed...

**Review by searchableguy — APPROVED** [review]
LGTM

File: unlock-app/src/config/app.ts [hunk 2]
```diff
@@ -45,10 +45,11 @@ const app = getAppConfig(env)
 
 export const config = {
   env,
-  wedlocksPublicKey: process.env.NEXT_PUBLIC_BASE64_WEDLOCKS_PUBLIC_KEY!,
+  wedlocksPublicKey:
```
Comment (searchableguy) [inline unlock-app/src/config/app.ts:48]: What is this for?

File: unlock-app/src/config/app.ts [hunk 3]
```diff
@@ -45,10 +45,11 @@ const app = getAppConfig(env)
...
+    'LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0NCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBbzdTYXhDZDg3YnQ0SFZ4anhXbkkNClFOTGFhZVdqb1ptbFNxcGhkL1JCbDNzNGQrcENnZHl3YUJBdGRqVENMQXBzakN6SzFqZWZLVWZmbXhqbW15NGUNCnFHRHN2ekZBazRUS1ZrM0g4ZTJrYnJpNHdoZjNaU1V4d2gyL2c2WWgrRzFuK1F2cWJ0ZGwvOVUvcnJ5SmR5NFENCmt6K2tJMzBsRk9aNHJCMHJ5bldvdDZuZGtISUVlMDFhQThzYWpSb1ZrZitDb1RFZ1hWNlowd2gzSlRNc01FUEENCk8rc2FGYkVaZlI4Y1lMNEVVanB1Ty9WUXZyZk1nVDRiQUxLQXVrV1hweGZWZDNWTVlNallQeU52KzUwRTFRR2UNClZYN0xtYUR6ZmhuQlhTbG4zU01mYncvTWl2cExoR3RlQ0NIN0JYaU8zb1hQUFpVd3ZIT3BzTm1OS2F6dEx4OG4NCmF3SURBUUFCDQotLS0tLUVORCBQVUJMSUMgS0VZLS0tLS0NCg==',
   stripeApiKey:
     process.env.NEXT_PUBLIC_STRIPE_KEY || 'pk_test_BHXKmScocCfrQ1oW8HTmnVrB',
-  ethPassApiKey: process.env.NEXT_PUBLIC_ETHPASS_KEY!,
+  ethPassApiKey: 'pk_live_Th4KfDH9DZw0cuHcWzyUFBgAeHEl0MoK',
```
Comment (searchableguy) [inline unlock-app/src/config/app.ts:52]: I know it's easily available from public site but pushing them to github makes it liable to get scrapped by some bot who is looking to spam + github automatically disables some keys it finds via secret scanning partner program.

Comment (julien51) [reply unlock-app/src/config/app.ts:52]: Ha! I can move that to 1passwird if that's an issue .. I'll do it

**Comment by julien51** [comment]
Closing in favor of smaller changes

**Pull request closed without merging**